src/index.ts

```diff
@@ -107,151 +108,106 @@ export interface CreateGraphOptionsSpec {
   edgeTable: string;
 }
 
-function fileToText(file: File): Promise<string> {
-  return new Promise((resolve, reject) => {
-    const reader = new FileReader();
-    reader.onload = (e) => {
-      if (e.target === null || typeof e.target.result !== 'string') {
-        throw new Error();
-      }
-      resolve(e.target.result);
-    };
-    reader.onerror = (e) => {
-      reject();
-    };
-
-    reader.readAsText(file);
-  });
-}
-
 class MultinetAPI {
-  public client: Client;
+  public axios: MultinetAxiosInstance;
 
   constructor(baseURL: string) {
-    this.client = new Client(baseURL);
+    this.axios = multinetAxiosInstance(axios.create({
+      baseURL,
+    }));
+
   }
 
   public logout() {
-    this.client.get('/user/logout');
+    this.axios.get('/user/logout');
```

Comment:
I think we should either implement this call in `multinetAxiosInstance`, or remove `logout` from the `MultinetAxiosInstance` interface.

Reply:
I will do the former.

Reply:
See d7cafdfb56d5a0dad75309be31b09396b01b133d.